roles/kafka_controller/tasks/kraft_tasks.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: get uuid
+  shell: "{{ binary_base_path }}/bin/kafka-storage random-uuid"
+  environment:
+    KAFKA_OPTS: "-Xlog:all=error -XX:+IgnoreUnrecognizedVMOptions"
+  register: uuid_key
+  run_once: true
+
+- name: get CLUSTER_ID
+  set_fact:
+    clusterid: "{{ uuid_key.stdout }}"
+
+- name: format meta directory
+  shell: "{{ binary_base_path }}/bin/kafka-storage  format -t {{ clusterid }} -c {{ config_base_path }}/etc/kafka/server.properties --ignore-formatted"
```

Comment:
Let's use the final property file variable here as well.
`kafka_controller.config_file`